### DISCUSSION TITLE
Question about SKpico2040DAC usage outside of C64

### DISCUSSION THREAD

**Discussion by jeff464** [frntc/SIDKick-pico#76] [Q&A]
I have a hardware DIY standalone MIDI synth that uses an original 6581 and I'm looking to replace it with a modern alternative. The SIDKick seems to be the ideal solution, but I had a few questions.

- Is the SKpico2040DAC what I'm looking for? I prefer a direct drop-in replacement without sourcing a Raspberry Pi, external wiring, audio jacks, etc.

- I don't have an original c64 and I just need a standard 8580 output without bells and whistles. Am I still able to set configurations/8580 settings?

- Should I concern myself with power settings, etc? Anything else I should know when using this outside of a c64 environment?

Thanks in advance.

**Comment by frntc** [comment]
In principle it would be easy for compile a firmware which just runs as a 8580 (without config tool and other extras). Also power is not an issue as the SKpico only uses 5V (6581 and 8580 differ in the secondary voltage which is unused here).

The only thing I'm a bit reluctant: the bus communication is tested to work in C64 and C128, and in principle there might be timings which are within the specs of the original SID, but not covered by the current firmware. Do you have any details on the synth? I assume it's nothing I could easily get hands on? :-)

**Comment by jeff464** [comment]
Thanks for the reply. That helps! Here's info about the synth: http://www.ucapps.de/midibox_sid.html

I'm not sure it says anything about timing differences, but that's way over my head anyway. ;)

Also, is PCBway the only way to make the SKpico2040DAC, or is something like JLCPCB an option as well?

**Reply by Kirtai** [reply]
Heh, I've been looking at that exact same project too :)
Hope it works.

**Reply by frntc** [reply]
I had a (quick) look at the webpage, unfortunately many links are dead (e.g. those to sellers of PCBs, some to the forum <- which also has a limited search functionality for non-registered users). Do you happen to have a link to the PCBs that are required for minimal setups? ... and to the firmware for the uC? In any case, it's certainly doable to get the SKpico running (but note: the ext-in is not supported)

PCBWay is currently the only option if you want to let them manufacture on your onw, but restore-store will have them soon as well.

**Reply by jeff464** [reply]
Here are a few links with info you might be looking for:

Base PCB:
http://www.midibox.org/dokuwiki/lib/exe/fetch.php?media=mb-6582:mb-6582_base_pcb_r2_color.pdf

SID pinout:
http://www.ucapps.de/mbhp/mbhp_sid_v3.pdf

Firmware:
http://www.ucapps.de/mios/midibox_sid_v2_046.zip

Hope this helps!

**Reply by frntc** [reply]
I had a quick look at the materials, I'd say from a technical perspective there's no reason why it should not work. But it still might be that some adjustments in timings might be necessary. If you'd give it a try, we can figure this out. Guessing from the PCB layout the larger SKpicos should also fit (although not every slot) to try things out.

**Comment by LouDnl** [comment]
Not wanting to hijack the discussion nor pushing my project, but my project is tested with SKPico and features a configurable midi device (finishing up the code). I hope to release the pcb in the coming weeks.
https://github.com/LouDnl/USBSID-Pico
